homeassistant/components/tailwind/binary_sensor.py

```diff
@@ -30,10 +30,10 @@ class TailwindDoorBinarySensorEntityDescription(BinarySensorEntityDescription):
 DESCRIPTIONS: tuple[TailwindDoorBinarySensorEntityDescription, ...] = (
     TailwindDoorBinarySensorEntityDescription(
```

Comment:
Would the device class problem fit now?

Reply:
Yeah that would work, except the translations wouldn't match I guess